src/os/Transaction.h

```diff
@@ -248,13 +254,13 @@ class Transaction {
 public:
   Transaction() = default;
 
-  explicit Transaction(ceph::buffer::list::const_iterator &dp) {
-    decode(dp);
-  }
-  explicit Transaction(ceph::buffer::list &nbl) {
-    auto dp = nbl.cbegin();
-    decode(dp);
-  }
+  //  explicit Transaction(ceph::buffer::list::const_iterator &dp) {
```

Comment:
Reviewer feedback requested:

I've deleted these helper functions to decode Transactions because there are now two decode functions (with one or two buffers) and I wanted code to be more explicit about which decode function to use. These functions were not being used by the current code.

Just want to check that there isn't a coding convention for having these type of decode helper functions?